zerver/lib/email_notifications.py

```diff
@@ -686,26 +687,58 @@ def get_onboarding_email_schedule(user: UserProfile) -> Dict[str, timedelta]:
         # of the user's inbox when the user sits down to deal with their inbox,
         # or comes in while they are dealing with their inbox.
         "followup_day2": timedelta(days=2, hours=-1),
+        "onboarding_zulip_guide": timedelta(days=4, hours=-1),
     }
 
     user_tz = user.timezone
     if user_tz == "":
         user_tz = "UTC"
     signup_day = user.date_joined.astimezone(zoneinfo.ZoneInfo(user_tz)).isoweekday()
 
+    # User signed up on Tuesday
+    if signup_day == 2:
+        # Send followup_day2 on Thursday
+        # Send onboarding_zulip_guide on Friday
+        onboarding_emails["onboarding_zulip_guide"] = timedelta(days=3, hours=-1)
+
+    # User signed up on Wednesday
+    if signup_day == 3:
+        # Send followup_day2 on Friday
+        # Send onboarding_zulip_guide on Monday
+        onboarding_emails["onboarding_zulip_guide"] = timedelta(days=5, hours=-1)
+
     # User signed up on Thursday
     if signup_day == 4:
         # Send followup_day2 on Friday
+        # Send onboarding_zulip_guide on Monday
         onboarding_emails["followup_day2"] = timedelta(days=1, hours=-1)
 
     # User signed up on Friday
     if signup_day == 5:
         # Send followup_day2 on Monday
+        # Send onboarding_zulip_guide on Tuesday
         onboarding_emails["followup_day2"] = timedelta(days=3, hours=-1)
 
     return onboarding_emails
 
 
+def get_onboarding_zulip_guide_context(realm: Realm) -> Tuple[Any, str]:
+    for realm_type, realm_type_details in Realm.ORG_TYPES.items():
+        if realm_type_details["id"] == realm.org_type:
+            organization_type_in_template = realm_type
+
+            # There are two education organization types that receive the same email
+            # content, so we simplify to one shared template context value here.
+            if organization_type_in_template == "education_nonprofit":
+                organization_type_in_template = "education"
+
+            return (realm_type_details["onboarding_zulip_guide_url"], organization_type_in_template)
+
+    # We don't expect to get here as realm.org_type should match an id in Realm.ORG_TYPES,
+    # but the fall through case would be to not send the onboarding_zulip_guide email.
+    return (None, "")  # nocoverage
```

Comment:
We probably should have an `assert` or log an exception or something like that.

Reply:
Added a `logging.error`, but left the `# nocoverage` comment.

Reply:
Added test coverage as well.